src/app/core/services/database.service.ts

```diff
@@ -19,13 +19,16 @@ export class DatabaseService {
 
   async getUserProfile(id: any) {
     return await this.supabaseClient
-      .from('user_profile')
-      .select('first_name, last_name, about, user_image_path')
-      .eq('user_id', id);
+      .from('users')
+      .select('id, first_name, last_name, about, profile_image')
+      .eq('id', id);
```

Comment:
can you fix naming and typing i.e. `userId: string` for example

Reply:
done